src/app/configs/page.tsx

```diff
@@ -0,0 +1,12 @@
+import { UserButton } from '@clerk/nextjs';
+
+import styles from './styles.module.css';
+
+export default function Configs() {
+  return (
+    <div className={styles.container}>
+      <h1 className={styles.title}>Configs</h1>
```

Comment:
```suggestion
    <div className={styles.configsContainer}>
      <h1 className={styles.configsTitle}>Configs</h1>
```
Acho que é uma boa ideia adotar nomes únicos pra classes, deixa muito mais fácil pra realizar manutenção quando o projeto escalar.

Reply:
faz sentido, vou alterar aqui

Reply:
Done